fairworkflows/fairworkflow.py

```diff
@@ -173,10 +182,19 @@ def draw(self, filepath):
         different renderings of the visualization using Graphviz library. The
         .dot.png file is one of those renderings.
         """
+        try:
+            import graphviz
+        except ImportError as m:
+            warn('Cannot produce visualization of RDF, ' + str(m))
```

Comment:
Since we are not including graphviz in requirements.txt (which I agree with) should this warning indicate what version of graphviz should be installed by the user? Or at least indicate what version it is known to work with

Reply:
Agree, fixed that!